coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -0,0 +1,35 @@
+from coalib.results.result_actions.ShowPatchAction import ShowPatchAction
+from coalib.results.result_actions.ResultAction import ResultAction
+import os.path
+
+
+class IgnoreResultAction(ResultAction):
+
+    SUCCESS_MESSAGE = "Result ignored successfully."
+
+    is_applicable = staticmethod(ShowPatchAction.is_applicable)
```

Comment:
I should make my own method `is_applicable` if that so, What should I test in it?